server/src/test/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplateControllerTest.java

```diff
@@ -361,6 +361,47 @@ void testGetByTitle() {
         assertContentEqualsEntity(templateTwo, response.getBody().get().get(1));
     }
 
+    @Test
+    void testFindDoesNotGrabReviewsForNonAdmin() {
+        final MemberProfile memberOne = createADefaultMemberProfile();
+        final FeedbackTemplate template = saveDefaultFeedbackTemplate(memberOne.getId());
+        final FeedbackTemplate templateTwo = saveAnotherDefaultFeedbackTemplate(memberOne.getId());
+        saveReviewFeedbackTemplate(memberOne.getId());
+
+        final HttpRequest<?> request = HttpRequest.GET("/?")
+                .basicAuth(memberOne.getWorkEmail(), RoleType.Constants.MEMBER_ROLE);
+        final HttpResponse<List<FeedbackTemplateResponseDTO>> response = client.toBlocking()
+                .exchange(request, Argument.listOf(FeedbackTemplateResponseDTO.class));
+
+        assertEquals(HttpStatus.OK, response.getStatus());
+        assertTrue(response.getBody().isPresent());
+
+        // Only contains the two non-reviews
+        assertEquals(2, response.getBody().get().size());
+        assertContentEqualsEntity(template, response.getBody().get().get(0));
+        assertContentEqualsEntity(templateTwo, response.getBody().get().get(1));
+    }
+
+    @Test
+    void testFindReviewsForAdmin() {
```

Comment:
This test saves 3 templates, one of which is a review template, then retrieves all templates as an ADMIN and validates that all three were returned.